src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -28,12 +37,28 @@ public CreateCaseCallbackService(
      * @return Either TBD - not yet implemented
      */
     public Either<List<String>, ExceptionRecord> process(CaseDetails caseDetails, String eventId) {
-        return validator
-            .mandatoryPrerequisites(() -> isCreateCaseEvent(eventId))
+        return assertAllowToAccess(caseDetails, eventId)
             .flatMap(theVoid -> validator
                 .getValidation(caseDetails)
                 .toEither()
                 .mapLeft(Seq::asJava)
             );
     }
+
+    private Either<List<String>, Void> assertAllowToAccess(CaseDetails caseDetails, String eventId) {
+        return validator.mandatoryPrerequisites(
+            () -> isCreateCaseEvent(eventId),
+            () -> getServiceConfig(caseDetails).map(item -> null)
+        );
+    }
+
+    private Validation<String, ServiceConfigItem> getServiceConfig(CaseDetails caseDetails) {
+        return hasServiceNameInCaseTypeId(caseDetails).flatMap(service -> Try
```

Comment:
we should be retrieving transformation url based on jurisdiction and not service name as CCD case detail object would know about jurisdiction and not service(internal to orchestrator)

```
 - service: sscs
      jurisdiction: SSCS

```

Reply:
I am aware you can still use service name but when it can be simply done using jurisdiction look up then we should use it :)

Reply:
and what about finrem service? jurisdiction is divorce

Reply:
Good point so we can’t do it per jurisdiction.